### PR TITLE
Migrate some docs from the GitHub wiki into the repository

### PR DESCRIPTION
**Pull request by SpaceManiac**
Moved to the code docs:
- Add AnturK's "GC Issues" guide
- Add Mothblocks's "Unit Tes documentation and guide
- Add non-outdated content of neersighted's "Common Variables and Functions" guide

Moved to CONTRIBUTING.md:
- Add bobbahbrown's "Issue Manager" guidelines

Other CONTRIBUTING.md changes:
- Inline the Maintainer Guidelines, instead of being in their own file
- Correct links, improve formatting, and do a light editing pass

The rest of the contents of the GitHub Wiki have been migrated to the tgstation13.org wiki.

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
stupid webhook

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -425,27 +475,27 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 
 #### Use
 * Bitwise AND - '&'
-	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
+	* Should be written as `variable & CONSTANT` NEVER `CONSTANT & variable`. Both are valid, but the latter is confusing and nonstandard.
```
Comment (LemonInTheDark) [inline .github/CONTRIBUTING.md:478]: I'm not sure this is a good change, although field and flag aren't really explained here, maybe they should be

File: code/__DEFINES/misc.dm [hunk 2]
```diff
@@ -185,10 +185,15 @@ GLOBAL_LIST_EMPTY(bloody_footprints_cache)
...
+/// Get the turf that `A` resides in, regardless of any containers.
+///
+/// Use in favor of `A.loc` or `src.loc` so that things work correctly when
+/// stored inside an inventory, locker, or other container.
 #define get_turf(A) (get_step(A, 0))
 
-//Same as above except gets the area instead
+/// Get the ultimate area of `A`, similarly to [get_turf].
+///
+/// Use instead of `A.loc.loc`.
```
Comment (LemonInTheDark) [inline code/__DEFINES/misc.dm:196]: Does not follow code doc guidelines, see https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
2 very minor things, F for the wiki

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (SpaceManiac) [reply .github/CONTRIBUTING.md:478]: which of "bitfield" and "bitflag" is the variable and which is the constant is not obvious even to me; I am only able to guess what the author meant based on what they said was correct

File: code/__HELPERS/mobs.dm [hunk 3]
```diff
@@ -250,7 +250,10 @@ GLOBAL_LIST_EMPTY(species_list)
 	return ..()
 
 
-///Timed action involving one mob user. Target is optional.
+/// Timed action involving one mob user. Target is optional.
+///
+/// Checks that `user` does not move, change hands, get stunned, etc. for the
+/// given `delay`. Returns `TRUE` on success or `FALSE` on failure.
```
Comment (LemonInTheDark) [inline code/__HELPERS/mobs.dm:256]: This should be updated

File: code/__HELPERS/unsorted.dm [hunk 4]
```diff
@@ -141,7 +141,10 @@ Turf and target are separate in case you want to teleport some distance from a t
 
 	return destination
 
-/proc/getline(atom/M,atom/N)//Ultra-Fast Bresenham Line-Drawing Algorithm
+/// Get a list of turfs in a line from `M` to `N`.
+///
+/// Uses the ultra-fast [Bresenham Line-Drawing Algorithm](https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm).
```
Comment (LemonInTheDark) [inline code/__HELPERS/unsorted.dm:146]: Ditto

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (LemonInTheDark) [reply .github/CONTRIBUTING.md:478]: That's true. Might be worth integrating this with the tg wiki's guide to bitflags from the yee olden days, but that's something to do later

**Comment by SpaceManiac** [comment]
@Cyberboss 

> Cyberboss added the Merge Conflict label yesterday 

stop reviewing my PRs while high

**Review by LemonInTheDark — APPROVED** [review]

**Comment by Cyberboss** [comment]
It was because there's no other way to clear the stupid linter failure other than the PR being merged with master

**Pull request merged**